### ISSUE TITLE
Add helper function to check whether a field name is a valid JSKOS field

### ISSUE THREAD

**Issue by stefandesu** [gbv/jskos-tools#11]
`isKnownJSKOSFieldForThisTypeOfEntity` or something similar.

**Comment by nichtich** [comment]
Is this needed to simplify code elsewhere? Looks like this better should be part of jskos-validate.

**Comment by stefandesu** [comment]
I don't think so. To be honest, I don't remember where we thought we would need this at all. jskos-validate seems to be the right place for this.
